cruft/_commands/utils/generate.py

```diff
@@ -36,6 +36,8 @@ def cookiecutter_template(
 
     repo.head.reset(commit=commit, working_tree=True)
 
+    if repo.working_dir is None:
```

Comment:
```suggestion
    assert repo.working_dir is not None
```

Reply:
That's what I did originally, but then bandit raised a warning that assert statements may be ignored during runtime.

Reply:
```
+ poetry run bandit -r cruft/
[main]  INFO    profile include tests: None
[main]  INFO    profile exclude tests: None
[main]  INFO    cli include tests: None
[main]  INFO    cli exclude tests: None
[main]  INFO    running on Python 3.8.12
Run started:2022-04-20 16:51:04.089010

Test results:
>> Issue: [B101:assert_used] Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.
   Severity: Low   Confidence: High
   Location: cruft/_commands/utils/generate.py:39:4
   More Info: https://bandit.readthedocs.io/en/latest/plugins/b101_assert_used.html
38
39          assert repo.working_dir is not None
40          context = _generate_output(cruft_state, Path(repo.working_dir), cookiecutter_input, output_dir)
```

@samj1912, do you still prefer the assert approach?

Reply:
Happy to add either a bandit or mypy ignore statement then :) https://bandit.readthedocs.io/en/latest/config.html#exclusions

Reply:
Thanks! This is my first time seeing bandit.